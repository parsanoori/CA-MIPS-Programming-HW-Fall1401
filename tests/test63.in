36
450761422
511675913
306569703
829463512
1055142653
823116558
621240611
389252281
733712456
173841101
20741910
536804899
354441851
103387872
702294662
437112469
911284476
479927613
171462680
29357613
604672226
851566635
961472877
824750302
976366353
386358466
457589882
551722952
49015948
50481000
974277264
82336597
827767941
433961603
822211257
756818282