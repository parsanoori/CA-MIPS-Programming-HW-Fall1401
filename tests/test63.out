20741910
29357613
49015948
50481000
82336597
103387872
171462680
173841101
306569703
354441851
386358466
389252281
433961603
437112469
450761422
457589882
479927613
511675913
536804899
551722952
604672226
621240611
702294662
733712456
756818282
822211257
823116558
824750302
827767941
829463512
851566635
911284476
961472877
974277264
976366353
1055142653